thirdparty/github.go

```diff
@@ -252,12 +255,34 @@ func getGithubClientRetry(token, caller string) *http.Client {
 	)
 }
 
+func GetInstallationToken(ctx context.Context, owner, repo string, opts *github.InstallationTokenOptions) (string, error) {
```

Comment:
Doc comment for exported functions.

Although, if this function is just being used in this one package, I would instead make this unexported.